### DISCUSSION TITLE
Turn off download video popup

### DISCUSSION THREAD

**Discussion by m-pathania** [subhra74/xdm#786] [Q&A]
I rarely download videos and the popup that says download video becomes very annoying. But if I download any files I use XDM for that. So I was wondering if there is any way to turn off the download video popup and only show pop up when I download a file?

**Comment by RyanMaxiemus** [comment]
In case you haven't found an answer yet:

I'm on Ubuntu 22.04, however, I'm sure the process is similar on other OSs. 

1. Go to the XDM app on your OS, not the extension in your browser. 
2. Click "Tools"
3. Click "Options..."
4. Click the checkbox that says "Disable floating 'DOWNLOAD VIDEO' popup"
5. Back out of the settings and that should be it.

Screenshots For Reference:

Step 2-3:
![XDM Settings Location (2022-06-09)](https://user-images.githubusercontent.com/35439558/172980459-512a4814-7398-4cd0-8f4b-cca34156647e.jpg)

Step 4:
![XDM Settings Panel (2022-06-09)](https://user-images.githubusercontent.com/35439558/172981556-b0d257f4-dfd0-424c-83b3-6cc07323caef.jpg)

Hope that helps.
